packages/cli/src/commands/session.ts

```diff
@@ -16,7 +16,7 @@ const register: (program: Command) => Command = (program) => program
 
 function action(options: Command): void {
   const { network, from, timeout, close, expires } = options;
-  session({ network, from, timeout, close, expires });
+  session({ network, from, timeout: <number> timeout, close, expires: <number> expires });
```

Comment:
This cast is interesting. 
A small curiosity (outside of the scope of this port): What happens if the user inputs an invalid number for timeout?

Reply:
I'm not sure what would happen, but I think that typescript *tries* to cast the variable if possible. If not, it leaves it as it is. OTOH, speaking about what's the current zos behaviour, I'm pretty sure that the option will be ignored.